src/config/index.js

```diff
@@ -358,8 +359,8 @@ module.exports = {
           return `/api/scoring/batches/${batchCode}/quizzes/${id}`
         },
         students: {
-          list(batchCode, page, pageSize) {
-            return `/api/scoring/batches/${batchCode}/quizzes?page=${page}&size=${pageSize}`
+          list(batchCode, isPassedDeadline, page, pageSize) {
```

Comment:
wanted to differentiate student api and quiz list api even though the value is identical?

Reply:
Yes, intended for readability on the studentQuiz list's javascript part.
However not sure if this is the best practice.